### DISCUSSION TITLE
Run the same Dagster job with different run_configs in parallel

### DISCUSSION THREAD

**Discussion by bruk0** [dagster-io/dagster#24286] [Q&A]
Hi, I need to create a Dagster pipeline with embedded-elt(dlt). I am looking for an information about how to run the same Dagster pipeline (Dagster job) with different run_configs and schedules. 

And how to run that with different run_configs in parallel

**Comment by garethbrickman** [comment]
Check out this section in our docs for [Specifying runtime configuration](https://docs.dagster.io/concepts/configuration/config-schema#specifying-runtime-configuration)

**Comment by bruk0** [comment]
Thank you, @garethbrickman for your response. As far as I understood, the `execute_in_process()` works for Ops and `materialize()` - for Assets. 

In that case, how to define `materialize()` in `jobs=[]` in `Definitions` class?

Sorry, for asking stupid questions, I need to adapt Dagster for our needs, and there is no specific info or tutorials in docs.

**Reply by garethbrickman** [reply]
Jobs are distinct between ops and assets. See here for defining[ asset jobs](https://docs.dagster.io/concepts/assets/asset-jobs#asset-jobs).

Once your jobs are defined you can define [schedules](https://docs.dagster.io/concepts/automation/schedules#schedules) to run them.

**Reply by bruk0** [reply]
@garethbrickman here's a picture what use case I have and what I want to implement
![image](https://github.com/user-attachments/assets/83149992-0b50-4cd1-affe-1c0908e0e585)
